bin/eslint-github-init.js

```diff
@@ -68,14 +71,49 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // TODO: Keep tsconfig.json and tslint.json as files in the lib and copy it over, rather then keeping it here inline.
```

Comment:
TODO

Reply:
Actually the more I look at this, I think this is correct. I don't think we keep any other config as a file currently so I think I'll follow that path.

Reply:
Oh we do for prettier!